query_group (#8671)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/core-upgrade/03-upgrading-to-v1.12.md

## Adapter-specific features and functionalities

> ### Session configuration

Author: ```suggestion
**Coming soon**
### Session configuration
```

Author: Note for the reviewer, I removed **Coming soon** first then added it back as v1.12 is yet to be released. I can remove it if that was the incorrect thing to do

> **Coming soon**

Reviewer: ```suggestion
```

### Redshift

> - The Redshift adapter supports the `query_group` session parameter, allowing dbt to tag queries for Redshift Workload Manager routing and query logging. When configured in a profile, dbt sets `query_group` when opening a connection and the value applies for the duration of that session. You can also configure `query_group` at the model level to temporarily override the default for a specific model, and dbt reverts the value at the end of model materialization. For more information, see [Redshift configurations](/reference/resource-configs/redshift-configs#session-configuration).

Reviewer: ```suggestion
- Added support for the `query_group` session parameter, allowing dbt to tag queries for Redshift Workload Manager routing and query logging. When configured in a profile, dbt sets `query_group` when opening a connection and the value applies for the duration of that session. You can also configure `query_group` at the model level to temporarily override the default value for a specific model, and dbt reverts the value at the end of model materialization. For more information, see [Redshift configurations](/reference/resource-configs/redshift-configs#session-configuration).
```

**Coming soon**

### Redshift

> - The Redshift adapter supports the `query_group` session parameter, allowing dbt to tag queries for the Redshift Workload Manager (WLM) routing and query logging. When configured, dbt sets `query_group` when opening a connection, and the value applies for the duration of that session. The `query_group` parameter can be configured at either the profile level or the model level, enabling a default value for all executions or an override for specific models.

Reviewer: I'm not sure if "When configured, dbt sets `query_group` when opening a connection, and the value applies for the duration of that session." applies to both profile and model level configuration. Can you please verify the technical accuracy of the suggested change:
```suggestion
- The Redshift adapter supports the `query_group` session parameter, allowing dbt to tag queries for Redshift Workload Manager routing and query logging. When configured in a profile, dbt sets `query_group` when opening a connection and the value applies for the duration of that session. You can also configure `query_group` at the model level to temporarily override the default for a specific model, and dbt reverts the value at the end of model materialization. For more information, see [Redshift configurations](/reference/resource-configs/redshift-configs#session-configuration).
```

Author: Hiya @luna-bianca 

I re-reviewed the repo and updated this and the upgrade section. Thank you for calling this out.

Kind regards
Natalie

>     See [Session configuration](/reference/resource-configs/redshift-configs#session-configuration) for more information.

Reviewer: ```suggestion
```
